--- a/modules/shepherd/service.scm
+++ b/modules/shepherd/service.scm
@@ -1059,6 +1059,8 @@
             ((? channel? notification)
              ;; We won the race: we're responsible for starting SERVICE
              ;; and sending its running value on NOTIFICATION.
+             (local-output (l10n "Starting service ~a...")
+                           (service-canonical-name service))
              (let ((running
                     (catch #t
                       (lambda ()
